lib/rubocop/ast/processed_source.rb

```diff
@@ -159,6 +159,20 @@ def line_indentation(line_number)
           .length
       end
 
+      def tokens_within(source_range)
```

Comment:
Last thing: maybe we should accept a `source_range` or a `node`?
I try to avoid these for fast methods, but here a single `if` won't be noticeable. WDYT?